cpp/ql/src/experimental/Security/CWE/CWE-1126/DeclarationOfVariableWithUnnecessarilyWideScope.qhelp

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Using variables with the same name is dangerous. However, such a situation inside the while loop can lead to a violation of the accessibility of the program. Requires the attention of developers.</p>
```

Comment:
> However, such a situation inside the while loop can lead to a violation of the accessibility of the program.

Please could you rephrase this sentence, I'm not sure what you mean by "the accessibility of the program".

There's a similar problem in the query `@description` as well.

Reply:
sorry for these mistakes.
I am working on my English, but I am ashamed in front of you every time.

I meant the impact of this error on the creation of an infinite loop exhausting resources.
is it sufficient to replace `accessibility` with` availability`?

Reply:
I think you described it clearly in your reply (above), but neither word (`accessibility` or `availability`) conveys this meaning.  Perhaps:
```
However, such a situation inside the while loop can create an infinite loop exhausting resources.
```

Reply:
thank you